waterloop-site/src/components/SustainableTech/NavBar/SideBar.tsx

```diff
@@ -118,86 +115,52 @@ const ToggleLine = styled.div`
   }
 `;
 
-type MyProps = {
-  sidebarOpen: boolean;
-  handleClickSidebar: (open: boolean) => void;
-};
+const Sidebar: React.FC = () => {
+  const [sidebarOpen, setSidebarOpen] = useState<boolean>(false);
+
+  const navLinks = [
+    { id: 'IDlocation1', name: 'Home' },
+    { id: 'IDlocation2', name: 'About' },
+    { id: 'IDlocation3', name: 'Team' },
+    { id: 'IDlocation4', name: 'Schedule' },
+    { id: 'IDlocation6', name: 'FAQ' },
```

Comment:
According to Figma, these should be all capitalized. https://www.figma.com/file/NhOtZxMSK7BTNsOrNuggTE/Sustainable-Tech-Event-Designs?node-id=82%3A209